tests/release/pipelines/rh_advisories.go

```diff
@@ -97,6 +97,9 @@ var _ = framework.ReleasePipelinesSuiteDescribe("e2e tests for rh-advisories pip
 			if err = managedFw.AsKubeDeveloper.TektonController.StorePipelineRun(pipelineRun.Name, pipelineRun); err != nil {
 				GinkgoWriter.Printf("failed to store PipelineRun %s:%s: %s\n", pipelineRun.GetNamespace(), pipelineRun.GetName(), err.Error())
 			}
+			if err = managedFw.AsKubeDeveloper.TektonController.StoreTaskRunsForPipelineRun(managedFw.AsKubeAdmin.CommonController.KubeRest(), pipelineRun); err != nil {
```

Comment:
Don't we want this in all of our tests?

Reply:
We need to add it to all of the test.  Just want to start from one test case in case any error happens. Is that ok?

Reply:
Yep